cds_ils/migrator/api.py

```diff
@@ -61,10 +61,11 @@ def import_documents_from_dump(sources, source_type, eager, include):
                         )
 
 
-def import_record(dump, model, pid_provider, legacy_id_key="legacy_recid"):
+def import_record(dump, model, pid_provider,
+                  legacy_id_key="legacy_recid", mint_id=None):
     """Import record in database."""
     record = CDSRecordDumpLoader.create(
-        dump, model, pid_provider, legacy_id_key
+        dump, model, pid_provider, legacy_id_key, mint_id
```

Comment:
what is a `mint_id`?

Reply:
same question

Reply:
It's used to pass the id we want to mint for that record, since we were not doing it. Already fixed in a better way by @kprzerwa in another PR, so I reverted these changes.